VAMobile/src/components/CommonErrorComponents/CallHelpCenter.tsx

```diff
@@ -88,7 +88,7 @@ const CallHelpCenter: FC<CallHelpCenterProps> = ({
               </Box>
```

Comment:
Could potentially be simplified moving this to the `primaryButton` prop to AlertWithHaptics.